corehq/apps/domain/tests/test_delete_domain.py

```diff
@@ -41,8 +39,10 @@ def _create_data(self, domain_name, i):
     def setUp(self):
         self.domain = Domain(name="test", is_active=True)
         self.domain.save()
+        self.domain.convert_to_commtrack()
         self.domain2 = Domain(name="test2", is_active=True)
         self.domain2.save()
+        self.domain2.convert_to_commtrack()
```

Comment:
A lot of the test failures were of this nature - tests took place on a non-commtrack test domain, but depended on commtrack-only functionality.  It seemed to make more sense to be explicit about being a commtrack domain than to work around it.